llama_hub/kibela/base.py

```diff
@@ -0,0 +1,104 @@
+"""LLama Kibela Reader"""
+from typing import Dict, List, Optional, TypeVar, Generic
+from gql import Client, gql
```

Comment:
sorry i think you might have to do a lazy import here for tests to pass

Reply:
Thank you. I fixed it.